overlord/snapstate/snapmgr.go

```diff
@@ -363,21 +358,7 @@ func Manager(st *state.State) (*SnapManager, error) {
 	return m, nil
 }
 
-func diskAliasTask(t *state.Task) bool {
-	// TODO: aliases v2!
-	kind := t.Kind()
-	return kind == "setup-aliases" || kind == "remove-aliases" || kind == "alias"
-}
-
 func (m *SnapManager) blockedTask(cand *state.Task, running []*state.Task) bool {
```

Comment:
Given that we remove the content of this function, do we need it at all anymore? i.e. could we remove it (and the  SetBlocked(m.blockedTask) in 353).